docs/ibexa_engage/ibexa_engage.md

```diff
@@ -0,0 +1,85 @@
+---
+description: Ibexa Engage helps to engage audiences and collect valuable data by using interactive content.
+---
+
+# Ibexa Engage
+
+[[= product_name_engage =]] is a data collection tool. It gives you the ability to use the [Qualifio](https://qualifio.com/) tools to engage your audiences.
+You can use interactive content to build relationships and collect important data, for example, a list of recent orders, or personal information about customers.
+
+## Request access
+
+To use [[= product_name_engage =]], you must make arrangements with [[= product_name =]] to define the initial configuration, and then get and set up user account.
+
+1\. Go to the Back Office.
+
+2\. On the left panel, go to **Ibexa Engage**.
+
+3\. On the welcome page, provide the required information:
+
+- First and last name
+- Email address
+
+4\. Select the **I have read and agree to the Terms and Conditions** checkbox, and then click **Send request**.
+
+![Welcome page - form](img/welcome_page_form.png "Welcome page - form")
+
+[[= product_name =]] creates and provides user account. An invitation link is sent during the setup process.
+Once the registration and configuration process is completed, the welcome page displays information about campaign tools available with [[= product_name_engage =]] and link to [Qualifio documentation](https://support.qualifio.com/hc/en-us/categories/202280638-Campaigns) where you can get information about creating and managing campaigns.
+
+![Welcome page - active account](img/welcome_page_activated.png "Welcome page - active account")
+
+Click **Manage campaigns in Qualifio** button to open Qualifio page, where you can create and manage campaigns.
+
+![Qualifio page](img/qualifio_page.png "Qualifio page - campaigns manager")
+
+## Campaign
+
+A campaign is a collection of ideas that the user can customize and divide into steps.
+
+It consists of a welcome screen, a form step, an interaction element, and an exit screen.
+
+You can use Qualifio's existing templates and interactive elements, such as quizzes, pools, and forms, to create visually appealing, customized campaigns.
+Qualifio configures the backgrounds, themes, and designs.
+
+Use campaign to interact with viewers, boost ad impressions, or gather cookies from targeted audiences.
+
+Each campaign comes with a unique ID - string of numbers, that the platform automatically defines at the creation. Campaign ID is a reference to the campaign in the applications, services and data connections.
+
+You can set up prizes for customer when an email or another required pool is filled.
```

Comment:
Prizes don't apply to all campaigns. I will remove this part and leave it to Qualifio documentation. Same for the line below

Reply:
I would keep it (I've just updated it a bit). We say "you can", it doesn't mean that it applies to all campaigns.

Reply:
I think it's more important in the Campaign definition to mention about anonymous vs identified campaign, because it's related to data collection and all campaign types. One sentence could be sufficient.
Source: https://developers.qualifio.com/docs/engage/concepts#anonymous--identified-campaigns